modules/openfast-library/src/FAST_Subs.f90

```diff
@@ -10212,10 +10218,14 @@ SUBROUTINE FAST_RestoreFromCheckpoint_T(t_initial, n_t_global, NumTurbines, Turb
    ! deal with files that were open:
    IF (Turbine%p_FAST%WrTxtOutFile) THEN
       CALL OpenFunkFileAppend ( Turbine%y_FAST%UnOu, TRIM(Turbine%p_FAST%OutFileRoot)//'.out', ErrStat2, ErrMsg2)
-      IF ( ErrStat2 >= AbortErrLev ) RETURN
       CALL SetErrStat(ErrStat2, ErrMsg2, ErrStat, ErrMsg, RoutineName )
-      CALL WrFileNR ( Turbine%y_FAST%UnOu, '#Restarting here')
-      WRITE(Turbine%y_FAST%UnOu, '()')
+      IF ( ErrStat2 >= AbortErrLev ) RETURN
+      if (present(silent)) then
+         if (silent) then
+            CALL WrFileNR ( Turbine%y_FAST%UnOu, '#Restarting here')
```

Comment:
I was expecting this to be written when **not** silent. Seems counterintuitive to write extra text when silent is true.

Reply:
Whoops!  I shouldn't commit code in a hurry... Thanks for catching that.